sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 12.0.1-beta.2 (2020-01-15)
```

Comment:
Should this be changed to `12.0.1-beta.1`?

Reply:
Yeah, I'll change it once we decide if we need to release again or not.

Reply:
resolved